libs/aws/tests/integration_tests/chat_models/test_standard.py

```diff
@@ -22,10 +21,6 @@ def chat_model_params(self) -> dict:
     def standard_chat_model_params(self) -> dict:
         return {"temperature": 0, "max_tokens": 100}
 
-    @pytest.mark.xfail(reason="Not implemented.")
-    def test_stop_sequence(self, model: BaseChatModel) -> None:
-        super().test_stop_sequence(model)
```

Comment:
The `test_stop_sequence` integ tests should be kept, I verified that they're now working with your fix. Just replace the `@pytest.mark.xfail(reason="Not implemented.")` decorator with ` @pytest.mark.scheduled`